src/main/flight/feedforward.c

```diff
@@ -153,6 +154,10 @@ FAST_CODE_NOINLINE float feedforwardApply(int axis, bool newRcFrame, feedforward
         } else {
             setpointDelta[axis] = setpointDeltaImpl[axis];
         }
+
+        // apply feedforward transition
+        setpointDelta[axis] *= ffTransitionFactor > 0 ? MIN(1.0f, getRcDeflectionAbs(axis) * ffTransitionFactor) : 1;
+
```

Comment:
This has to be be done here too for the unit tests to pass.
https://github.com/betaflight/betaflight/blob/0b0455d7db8a759da9070565e0f5651b1c2b73f2/src/test/unit/pid_unittest.cc#L100-L105

Something like this
```   
    float feedforwardApply(int axis, bool newRcFrame, feedforwardAveraging_t feedforwardAveraging)
    {
        UNUSED(newRcFrame);
        UNUSED(feedforwardAveraging);
        const float ffTransitionFactor = pidGetFfTransition();
        float setpointDelta = simulatedSetpointRate[axis] - simulatedPrevSetpointRate[axis];
        setpointDelta *= ffTransitionFactor > 0 ? MIN(1.0f, getRcDeflectionAbs(axis) * ffTransitionFactor) : 1;
        return setpointDelta;
    }
```

Reply:
Thanks @klutvott123 have merged those unit test fixes.